README.md

```diff
@@ -25,29 +25,33 @@ This project contains the Chef setup for a Jenkins-based CI/CD server for Chef c
 ## Cookbooks:
 
 * t3-base (~> 0.2.0)
-* java (= 1.39.0)
-* jenkins (= 2.5.0)
+* t3-chef-vault (~> 1.0.0)
+* jenkins-chefci (~> 0.2.0)
+* java (= 1.50.0)
+* jenkins (= 5.0.2)
 * ssh_known_hosts (= 2.0.0)
 * chef-dk (= 3.1.0)
 * sudo
 * apt
 * git
+* #<Logger:0x00007fb7bfb4a298> () (Recommended but not required)
+* #<Logger:0x00007fb7bfb4a298> () (Suggested but not required)
+* Conflicts with #<Logger:0x00007fb7bfb4a298> ()
```

Comment:
@StephenKing That's a bug isn't it?

Reply:
yes, seems so. I installed the most recent version of `knife-cookbook-doc` and ran `knife cookbook doc .` once more and had the same output. Maybe you can check that on your machine, but I don't know anything I could do about it.